week-2/Python/Myles-Cherebin.py

```diff
@@ -0,0 +1,64 @@
+#prompt 1

```

Comment:
Hello
I like your coding. One things I noticed about your code is your coding style. I would improve it by adding comment at the beginning of each function if not to each line of code to make it simpler and easier to read. 
Its benefit is to gives the code reviewer or a programmer reading your code for a later use to be able to go easily go though your code without struggling to undersnad it. It will facilitate to read your program and understand all your assymtion and codes.

Reply:
Yes I agree. Posting a link in the slack that explains this a bit further